apps/web/cypress/tests/notifications.spec.ts

```diff
@@ -25,6 +25,16 @@ describe('Notification Templates Screen', function () {
     cy.getByTestId('category-label').contains('General');
   });
 
+  it('should show the create template dropdown', function () {
+    cy.intercept('**/notification-templates**').as('notificationTemplates');
+    cy.visit('/templates');
+    cy.wait('@notificationTemplates');
+
+    cy.getByTestId('create-template-btn').should('not.be.disabled').click();
+    cy.getByTestId('create-workfolow-all-templates').contains('All templates');
+    cy.getByTestId('create-workfolow-blank').contains('Blank workflow');
+  });
```

Comment:
right now we only check for `All templates` and `Blank workflow`, but in the near future we will initialize test with blueprints and then will check the most popular items in the dropdown